server/package.json

```diff
@@ -24,7 +23,6 @@
     "@azure/openai": "1.0.0-beta.10",
     "@datastax/astra-db-ts": "^0.1.3",
     "@google/generative-ai": "^0.7.1",
-    "@googleapis/youtube": "^9.0.0",
```

Comment:
No idea why this was ever server-side

Reply:
Came from merge on inital commit > 1 year ago
https://github.com/Mintplex-Labs/anything-llm/commit/27c58541bd555b8b9791ed6e2d8d7bdb252230ca